package.json

```diff
@@ -99,7 +99,8 @@
     "typescript": "4.3.5",
     "url-polyfill": "1.1.12",
     "url-search-params-polyfill": "8.1.0",
-    "vanilla-framework": "2.35.0"
+    "vanilla-framework": "2.35.0",
+    "yup": "0.32.9"
```

Comment:
Yup is great, but has there been any discussion regarding adding this as a dependency?

Also - is there a reason this couldn't go straight to master?

Reply:
We tend to use Yup with Formik (it's the recommended validation lib). No reason it couldn't go to master, but I needed it here.

Reply:
Don't get me wrong, I have nothing against yup (happy to use it, used it in the past), I was just making a point about thinking of merging code straight to master where possible. Such focused PR would also allow for a conversation across the team about adding such dependency.